lib/telemetry.js

```diff
@@ -60,5 +66,28 @@ export const startPingLoop = () => {
     point.tag('arch', arch())
     point.tag('deployment_type', DEPLOYMENT_TYPE)
     writeClient.writePoint(point)
-  }, 10 * 60 * 1000)
+    writeClient.flush().catch(handleFlushError)
+    await timers.setTimeout(10 * 60 * 1000) // 10 minutes
+  }
+}
+
+export const runMachinesLoop = async () => {
+  while (true) {
+    const point = new Point('machine')
+    point.intField('cpu_count', cpus().length)
+    point.intField('cpu_speed_mhz', cpus()[0].speed)
+    point.tag(
+      'cpu_brand',
+      /intel/i.test(cpus()[0].model)
+        ? 'intel'
+        : /amd/i.test(cpus()[0].model)
+          ? 'amd'
+          : 'unknown'
+    )
+    point.intField('memory_free_b', freemem())
```

Comment:
The amount of free memory changes often, but we are sampling the value only once per day. I think such data would most likely mislead us.

I slightly prefer to remove this field, but I don't feel too strongly about that.

Reply:
I agree, let's remove it for now